.github/workflows/release.yml

```diff
@@ -0,0 +1,94 @@
+name: Release pipeline
+on:
+  # Run when release is published
+  release:
+    types: [published]
+
+env:
+  SERVICES: context-box scheduler builder terraformer ansibler kube-eleven kuber frontend
+
+jobs:
+  build-and-publish:
+    name: Create a new release
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+
+      - name: Set release tag
+        run: |
+          R=${GITHUB_REF#"refs/tags/"}
+          echo "RELEASE=$R" >> $GITHUB_ENV
+
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
+      - name: Create and use docker context
+        run: |
+          docker context create claudie
+          docker context use claudie
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+        with:
+          endpoint: claudie
+
+      - name: Install kustomize
+        uses: imranismail/setup-kustomize@v2
+        with:
+          kustomize-version: 3.6.1
+
+      - name: Set up docker
+        uses: docker/login-action@v2
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build and push new images
+        run: |
+          ARR=( ${{ env.SERVICES }} )
+          for SERVICE in "${ARR[@]}"
+          do
+            echo "-----Building $SERVICE-----"
+            IMGTAG="ghcr.io/berops/claudie/$SERVICE:${RELEASE}" 
+            DOCKER_BUILDKIT=1 docker buildx build --push --platform linux/amd64,linux/arm64 --tag $IMGTAG -f ./services/$SERVICE/Dockerfile . 
+          done
+
+      - name: Set new tag to manifests
+        working-directory: manifests/claudie
+        run: |
+          ARR=( ${{ env.SERVICES }} )
+          for SERVICE in "${ARR[@]}"
+          do
+            kustomize edit set image "ghcr.io/berops/claudie/$SERVICE:${RELEASE}" 
+          done
+
+      - name: Create archives for manifests
+        working-directory: manifests/claudie
+        run: |
+          sudo apt install -y zip tar
+          tar -czvf ../../claudie.tar.gz .
+          zip -r ../../claudie.zip .
+
+      - name: Get checksums of the archives
+        run: |
+          touch claudie_checksums.txt
+          sha256sum claudie.tar.gz >> claudie_checksums.txt
+          sha256sum claudie.zip >> claudie_checksums.txt
+
+      - name: Add manifests to the release
+        uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          tag: ${{ github.ref }}
+          file: claudie.*
+          file_glob: true
+
+      - name: Add checksums file to the release
+        uses: svenstaro/upload-release-action@v2
+        with:
+          repo_token: ${{ secrets.GITHUB_TOKEN }}
+          tag: ${{ github.ref }}
+          file: claudie_checksums.txt
```

Comment:
this is great.

also, could we somehow instruct the users to also download the checksums fuke along with the assets, and verify the asset integrity with `sha256sum -c claudie_checksums.txt`? that's probably go into the Changelog file though.

Reply:
From what I have seen in other repos, `checksums.txt` was a standalone file alongside other releases and whoever wants to verify the assets, IMO already knows what to do.

Reply:
yeah, that's certainly a way to approach it.

another one is that we *want* our user's to actively verify what they download from us :)) and some user's might only have a very vague idea of what checksums are, surely we do want to point them in the right direction..

![image](https://user-images.githubusercontent.com/61180606/211825143-a5dfb53e-dc90-446d-b97e-7b7c0c6ba938.png)